substrate/frame/revive/src/lib.rs

```diff
@@ -303,6 +303,10 @@ pub mod pallet {
 		/// preventing replay attacks.
 		#[pallet::constant]
 		type ChainId: Get<u64>;
+
+		/// The ratio between the decimal representation of the native token and the ETH token.
+		#[pallet::constant]
+		type NativeToEthRatio: Get<u32>;
```

Comment:
Isn't handled by adding the network with the correct decimals to the wallet?

Reply:
in theory yes, but in practice no wallet support anything else than 18 decimals https://github.com/MetaMask/metamask-extension/issues/15168

Reply:
https://community.metamask.io/t/add-support-for-native-chain-tokens-not-18-decimals/25951

Reply:
Thanks!